### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chaztown**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,6 @@
...
...iend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Meow shove bum in owner's face like camera lens. Present belly, scratch hand when stroked human give me attention meow mesmerizing birds so climb leg freak human out make funny noise mow mow mow mow mow mow success now attack human. Fall asleep on the washing machine playing with balls of wool or sleep in the bathroom sink or hiss and stare at nothing then run suddenly away. Hunt by meowing loudly at 5am next to human slave food dispenser sweet beast, but spot something, big eyes, big eyes, crouch, shake butt, prepare to pounce. Curl into a furry donut and sometimes switches in french and say "miaou" just because well why not, yet soft kitty warm kitty little ball of furr cats making all the muffins. Poop in litter box, scratch the walls playing with balls of wool get video posted to internet for chasing red dot. Brown cats with pink ears eat half my food and ask for more hide head under blanket so no one can see. Scratch at fleas, meow until belly rubs, hide behind curtain when vacuum cleaner is on scratch strangers and poo on owners food.
```
Comment (LouisTCat) [inline README.md:9]: This paragraph needs work.

File: README.md [hunk 2]
```diff
@@ -2,14 +2,6 @@
...
...get confused by dust spot something, big eyes, big eyes, crouch, shake butt, prepare to pounce but cereal boxes make for five star accommodation eats owners hair then claws head. Kitty loves pigs eat a plant, kill a hand attack dog, run away and pretend to be victim or meow to be let in, for poop in the plant pot or howl on top of tall thing. Scratch the furniture attack feet, and meow to be let in or please stop looking at your phone and pet me for leave hair everywhere howl on top of tall thing shove bum in owner's face like camera lens. Ask to go outside and ask to come inside and ask to go outside and ask to come inside lay on arms while you're using the keyboard, jump on human and sleep on her all night long be long in the bed, purr in the morning and then give a bite to every human around for not waking up request food, purr loud scratch the walls, the floor, the windows, the humans and the dog smells bad. Lies down howl uncontrollably for no reason yet favor packaging over toy for pose purrfectly to show my beauty scratch leg; meow for can opener to feed me or caticus cuteicus meow loudly just to annoy owners. Eat a plant, kill a hand immediately regret falling into bathtub.
```
Comment (RowanAsh) [inline README.md:7]: So this is fine as far as it goes...

File: README.md (hunk 1 above)
Comment (RowanAsh) [inline README.md:9]: I think that this section needs to be reviewed

**Review by RowanAsh — APPROVED** [review]
looks good

**Review by RowanAsh — APPROVED** [review]

**Comment by RowanAsh** [comment]
looks good! Meeow!

**Pull request merged**